rubi/rubi/data/market.py

```diff
@@ -237,3 +253,160 @@ def get_offers(
             df = self.offer_query.query_offers(fields)
 
             return df
+
+    def get_trades(
+        self,
+        taker: Optional[str] = None,
+        from_address: Optional[str] = None,
+        pair_name: Optional[str] = None,
+        book_side: Optional[OrderSide] = OrderSide.NEUTRAL,
+        take_gem: Optional[
+            str
+        ] = None,  # TODO: not sure we really need this given the pair_name parameter
```

Comment:
I would say we either go with `pair_name` or we have the `take_gem` and `buy_gem` parameters.

As this is a lover level function on the data/market.py I suggest we remove pair_name and leave that abstraction to the client.